src/page/Order/components/MessageInput.tsx

```diff
@@ -2,6 +2,28 @@ import styled from '@emotion/styled';
 import type { OrderInfoValues } from '..';
 import { useFormContext } from 'react-hook-form';
 
+const MessageInput = () => {
+  const {
+    register,
+    formState: { errors },
+  } = useFormContext<OrderInfoValues>();
+
+  return (
+    <MessageInputContainer>
+      <Title>메시지 입력</Title>
+      <Textarea
+        placeholder="메시지를 입력해주세요."
+        {...register(`message`, {
+          required: { value: true, message: '메시지를 입력해주세요.' },
+        })}
+      ></Textarea>
```

Comment:
children이 없는 경우는 self-closing tag를 사용해주심 좋아용!
```suggestion
      <Textarea
        placeholder="메시지를 입력해주세요."
        {...register(`message`, {
          required: { value: true, message: '메시지를 입력해주세요.' },
        })}
      />
```